lib/main.dart

```diff
@@ -1,11 +1,19 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
 import 'package:firebase_core/firebase_core.dart';
+import 'chat_message.dart';
 import 'firebase_options.dart';
```

Comment:
If we don't push this file, it cannot be compiled. Shouldn't we commit it?

Reply:
I don't know! Do we want them to use ours or theirs? Because if we push, and they checkout, they get ours.

Reply:
in the dart file, you can remove the important information.

Reply:
it's not about information, it's about conflict that they may have, after using their own firebase config.

Reply:
Normally the file will be overwritten by the cli so it wouldn't be a problem, no?
would it?